projects/scion/workbench-application-platform/src/lib/spec/array.util.spec.ts

```diff
@@ -92,4 +92,38 @@ describe('Arrays', () => {
       expect(array).toEqual(['a', 'b', 'c', 'd', 'e']);
     });
   });
+
+  describe('Arrays.distinct', () => {
+
+    it('should remove duplicate string items', () => {
+      const array = ['a', 'a', 'b', 'c', 'd', 'e', 'c'];
+      expect(Arrays.distinct(array)).toEqual(['a', 'b', 'c', 'd', 'e']);
+    });
+
+    it('should remove duplicate complex items', () => {
```

Comment:
Please rename: `should remove duplicate objects by reference`

Reply:
Done